### ISSUE TITLE
Simulation set-up description

### ISSUE THREAD

**Issue by gemblerz** [gemblerz/gor-purdue#2]
Setting a simulation environment needs to be documented. That should include
* Version of any software tools
* Instruction of how to install the tools and configurations

It would be better if there is a single script that configures the environment.

**Comment by tony-sanguk** [comment]
I wrote two documents, `howtostart.md` and `howtotest.md`. They are about how to configure your computer to simulate our program. In addition, I made a note the version of our tools.

Nevertheless, I think we had better implement a simple script to configure this setting before we start simulating in earnest.

**Comment by gemblerz** [comment]
I agree. Let's keep this issue alive until we come up with a reasonable level of document. I think the reasonable level would be achieved when we use full stack of agent model to run something and have a document describing how to run it.
